### ISSUE TITLE
Possible integration with SCUMMVM?

### ISSUE THREAD

**Issue by neuromancer** [devinsmith/opendw#1]
I know this implementation of this game engine is still a work in progress, but I'm wondering if it makes sense to try to re-use the SCUMMVM implementation of common rpg game elements (actor, dialogs, etc) to accelerate the development progress.

**Comment by devinsmith** [comment]
Yes, that's probably possible and the game does use some sort of script "engine" which I'm still trying to figure out. Maybe this will be easier when I get further with the game script.
